cmd/hypper/hypper.go

```diff
@@ -27,7 +27,7 @@ func main() {
 	}
 
 	if err != nil {
-		logger.Debug(eycandy.Magenta("%v"), err)
```

Comment:
wait, this was on main and it didnt fail the ci?? How is that possible with the strictness of go??

Reply:
No because the package itself contained the typo in the packagename, so it was wrong but still correct code, fixed the packagename aswell.